react/components/DrawerMode.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+import { Drawer } from 'vtex.store-drawer'
+
+import MinicartIconButton from './MinicartIconButton'
+
+const DRAWER_CLOSE_ICON_HEIGHT = 57
```

Comment:
This value should be `58`, so the `height` would best fit the available space.
```suggestion
const DRAWER_CLOSE_ICON_HEIGHT = 58
```